### ISSUE TITLE
Create Keyspace Statement in DefaultSessionProvider Should Be Optional

### ISSUE THREAD

**Issue by j-potts** [outworkers/phantom#511]
For those of us who choose to use something outside of Phantom to initially create a given keyspace, there should be some way to initialize a Database without this query being executed: https://github.com/outworkers/phantom/blob/develop/phantom-connectors/src/main/scala/com/websudos/phantom/connectors/DefaultSessionProvider.scala#L40

This query can be quite dangerous, as it assumes a desired default implementation and not all environments may allow this permission for the Cassandra user. In our case, we do not wish to create the keyspace via Phantom at all, as we're simply using Phantom for the DSL and connector/session initialization.

**Comment by markglh** [comment]
+1

**Comment by felixt-cake** [comment]
👍

**Comment by sean-walsh** [comment]
+1

**Comment by j-potts** [comment]
Great, thanks @alexflav23 !

**Comment by alexflav23** [comment]
Hi @j-potts, @felixt-cake , @markglh,

This has now been fixed and will be available as part of `1.28.0`. You can set the `autoinit` to false when you do `ContactPoint.local.keySpace("space", autoinit = false)` which will skip the keyspace intialisation part completely.

In the next release, you will also be able to provide a custom `Keyspace` creation query using a typesafe builder using a syntax like the following. This should be available within the next 2 weeks if nothing interferes with current plans, schedules are unfortunately always more busy than we would like them to be.

```scala

val query = KeySpace

ContactPoint.local.keySpace(KeySpace("space").ifNotExists()
      .`with`(replication eqs NetworkTopologyStrategy)
      .and(durable_writes eqs true))
```

**Comment by alexflav23** [comment]
1.28.0 now available on Bintray. I guess we can mark this is as fixed, please let us know if you find any issues.
